config/viper_config.go

```diff
@@ -67,12 +67,22 @@ func (c *Config) fillDefaultValues() {
 		"pitaya.cluster.rpc.client.nats.connectiontimeout":      pitayaConfig.Cluster.RPC.Client.Nats.ConnectionTimeout,
 		"pitaya.cluster.rpc.client.nats.maxreconnectionretries": pitayaConfig.Cluster.RPC.Client.Nats.MaxReconnectionRetries,
 		"pitaya.cluster.rpc.client.nats.websocketcompression":   pitayaConfig.Cluster.RPC.Client.Nats.WebsocketCompression,
+		"pitaya.cluster.rpc.client.nats.reconnectjitter":        pitayaConfig.Cluster.RPC.Client.Nats.ReconnectJitter,
+		"pitaya.cluster.rpc.client.nats.reconnectjittertls":     pitayaConfig.Cluster.RPC.Client.Nats.ReconnectJitterTLS,
+		"pitaya.cluster.rpc.client.nats.reconnectwait":          pitayaConfig.Cluster.RPC.Client.Nats.ReconnectWait,
+		"pitaya.cluster.rpc.client.nats.pinginterval":           pitayaConfig.Cluster.RPC.Client.Nats.PingInterval,
+		"pitaya.cluster.rpc.client.nats.maxpingsoutstanding":    pitayaConfig.Cluster.RPC.Client.Nats.MaxPingsOutstanding,
```

Comment:
Can we add to `configuration.rst` document as well?

Reply:
good catch, done!